.claude/skills/release-chores.md

```
@@ -0,0 +1,277 @@
+- ✗ = Image does NOT exist
+
+**Images to verify**:
+
+1. **UBI Minimal**: `registry.access.redhat.com/ubi9/ubi-minimal:$NEW_UBI_VERSION`
+2. **Golang Builder**: `registry.ci.openshift.org/ocp/builder:rhel-9-golang-$GO_MINOR_VERSION-openshift-$NEW_VERSION`
+
+Do not prompt or stop on verification issues — continue and document results.
+
+### Step 5: Display Change Summary
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)